websites/views/blogs_daily_summary.view.lkml

```diff
@@ -1,27 +1,27 @@
 include: "//looker-hub/websites/views/blogs_daily_summary.view.lkml"
 
 view: +blogs_daily_summary {
-  measure: sessions {
-    label: "Sessions Count"
+  measure: sessions_sum {
+    label: "Sessions Sum"
     type: sum
-    sql: ${TABLE}.sessions;;
+    sql: ${sessions} ;;
```

Comment:
Additionally I'd set the dimensions with these names to hidden if they're meant to only be measures